src/main/scala/beam/agentsim/scheduler/BeamAgentScheduler.scala

```diff
@@ -207,55 +198,54 @@ class BeamAgentScheduler(val beamConfig: BeamConfig, stopTick: Double, val maxWi
 
     case Monitor =>
       log.debug(
-        s"\n\tnowInSeconds=$nowInSeconds,\n\tawaitingResponse.size=${awaitingResponse
-          .size()},\n\ttriggerQueue.size=${triggerQueue.size},\n\ttriggerQueue.head=${triggerQueue.headOption}\n\tawaitingResponse.head=$awaitingToString"
+        s"\n\tnowInSeconds=$nowInSeconds,\n\tawaitingResponse.size=${
+          awaitingResponse
+            .size()
+        },\n\ttriggerQueue.size=${triggerQueue.size},\n\ttriggerQueue.head=${triggerQueue.headOption}\n\tawaitingResponse.head=$awaitingToString"
```

Comment:
This log is very hard to read. Consider:
- replacing `\n` with a constant(or local variable) like: `val EOL = System.lineSeparator()`
- use scala multiline string to improve readability